### PR TITLE
Add ability to attach category to task on create task page

### PR DESCRIPTION
**Pull request by aaronw4**
# Description
This pull request will pull categories from the user and list them in a drop down menu on the create task form. It will also attach selected category to task being created and save it on the back end.

## Type of change

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge

# How Has This Been Tested?
Ran app on local machine and tested it with a copy of back-end on my own Heroku account.

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

File: src/Components/Home/Home.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ import Timer from '../Timer/Timer/Timer';
 class Home extends React.Component {
   constructor(props) {
     super(props)
-    console.log(props)
```
Comment (CodingCCarpenter) [inline src/Components/Home/Home.js:18]: Good call to delete the superfluous console.log!

File: src/Components/NewTask/Category.js [hunk 2]
```diff
@@ -1,77 +1,49 @@
-import React, { Component } from "react";
-import Downshift from "downshift";
+import React from "react";
+import {withRouter} from 'react-router-dom'
+import { connect } from "react-redux"
+import { Dropdown, DropdownButton } from 'react-bootstrap';
 
-
-class Category extends Component {
+class Category extends React.Component {
```
Comment (CodingCCarpenter) [inline src/Components/NewTask/Category.js:6]: Does this work better when we don't import Component directly?

**Review by CodingCCarpenter — APPROVED** [review]
This looks good!

File: src/Components/NewTask/Category.js (hunk 2 above)
Comment (aaronw4) [reply src/Components/NewTask/Category.js:6]: It is the same. I changed it from a class to a function and then back to a class. Didnt realize it was slightly different.

**Pull request merged**